Cannot find the KWin version in the latest master commit

Building from source code (as per instructions in the [readme](https://github.com/matinlotfali/KDE-Rounded-Corners?tab=readme-ov-file#how-to-build-from-source-code)) fails on OpenSUSE Tumbleweed. It used to work fine, but the latest change causes an error:
```
> cmake ..
-- KWin Version: 
grep: /usr/include/kwineffects.h: No such file or directory
-- KWinEffect Version: 
CMake Error at CMakeLists.txt:26 (find_package):
  By not providing "FindKWinEffects.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "KWinEffects", but CMake did not find one.

  Could not find a package configuration file provided by "KWinEffects" with
  any of the following names:

    KWinEffectsConfig.cmake
    kwineffects-config.cmake

  Add the installation prefix of "KWinEffects" to CMAKE_PREFIX_PATH or set
  "KWinEffects_DIR" to a directory containing one of the above files.  If
  "KWinEffects" provides a separate development package or SDK, be sure it
  has been installed.


-- Configuring incomplete, errors occurred!
```

I have all of the required dependencies for Plasma 6 that were mentioned in the readme. I did check and I don't have any `kwineffects.h` file, in /`usr/include` or elsewhere. Maybe it has a different name/location under OpenSUSE?

## System information:
- **OS**: OpenSUSE Tumbleweed
- **Plasma version**: 6.1.3
- **KDE Rounded Corners**: latest git (8464658e42087e20dca0b05cb99a293e6b15bcc4)

--- a/cmake/qtversion.cmake
+++ b/cmake/qtversion.cmake
@@ -1,5 +1,5 @@
 execute_process(
-        COMMAND sh -c "ldconfig -v 2>&1 | awk '$1 ~ \"libkwin.so\" {print $3}' | cut -d '.' -f 3-"
+        COMMAND sh -c "ldconfig -v 2>&1 | grep libkwin.so | rev | cut -d '.' -f -3 | rev"
         OUTPUT_VARIABLE KWIN_VERSION OUTPUT_STRIP_TRAILING_WHITESPACE
 )
 message("-- KWin Version: ${KWIN_VERSION}")

